Makefile

```diff
@@ -96,12 +100,28 @@ snyk: FORCE
 		snyk monitor --file=$${d} ; \
 	done
 
-docker-image: FORCE
```

Comment:
The docker workflow expects `make docker-image` to work. It will probably work if you changed https://github.com/CloudNativeDataPlane/cndp/blob/main/.github/workflows/docker.yml#L20 to `make oci-image`